### ISSUE TITLE
Regarding the nuScenes dataset, the Val set results and the Test set results

### ISSUE THREAD

**Issue by ZJUnewer** [AlmoonYsl/QTNet#13]
Excuse me, the test results of the validation set and the test set are similar. I would like to ask what changes were made when testing the set?

**Comment by AlmoonYsl** [comment]
> Excuse me, the test results of the validation set and the test set are similar. I would like to ask what changes were made when testing the set?

Like previous methods, we use the training and validation set for test set.

**Comment by ZJUnewer** [comment]
So shouldn’t the methods on the nuscene list be tested only on the test set?

**Comment by AlmoonYsl** [comment]
> So shouldn’t the methods on the nuscene list be tested only on the test set?

Sorry my answer is not clear, I mean we use training and validation set to train QTNet for test set.

**Comment by ZJUnewer** [comment]
Thanks, I will try

> > So shouldn’t the methods on the nuscene list be tested only on the test set?
> 
> Sorry my answer is not clear, I mean we use training and validation set to train QTNet for test set.

Thanks, I will try

**Comment by ZJUnewer** [comment]
> > So shouldn’t the methods on the nuscene list be tested only on the test set?
> 
> Sorry my answer is not clear, I mean we use training and validation set to train QTNet for test set.

Thanks, I will try.

**Comment by ZJUnewer** [comment]
Hello, when generating the memory_bank of query and box_result, do you use the training set only or the training set and verification set?

**Comment by AlmoonYsl** [comment]
> Hello, when generating the memory_bank of query and box_result, do you use the training set only or the training set and verification set?

We use the training set and validation set to generate the memory bank.
